qemu/tests/cfg/qsd_backup_pull.cfg

```diff
@@ -0,0 +1,82 @@
+- qsd_backup_pull:
+    only Linux
+    only filesystem
+    virt_test_type = qemu
+    type = qsd_backup_pull
+    qemu_force_use_drive_expression = no
+
+    # fleecing images full and inc
+    full_backup_image_stg1 = full
+    inc_backup_image_stg1 = inc
+    backup_images_stg1 = "full inc"
+    backing_full = stg1
+    backing_node_full = fmt_stg1
+    backing_inc = stg1
+    backing_node_inc = fmt_stg1
+    start_vm = no
+    storage_pools = default
+    storage_pool = default
+    storage_type_default = "directory"
+
+    image_size_stg1 = 2G
+    image_size_full = 2G
+    image_size_inc = 2G
+
+    image_format_stg1 = qcow2
+    image_format_full = qcow2
+    image_format_inc = qcow2
+
+    image_name_stg1 = stg1
+    image_name_full = full
+    image_name_inc = inc
+    source_images = "stg1"
+    rebase_mode = unsafe
+    dirty_bitmap_opt = x-dirty-bitmap
+
+    # conf of fleecing images exported,
+    # used for internal nbd server
+    nbd_export_writable = no
+    nbd_port_full = 10810
+    nbd_port_inc = 10811
+    nbd_export_name_full = nbd_full_image
+    nbd_export_name_inc = nbd_inc_image
+
+    # conf of nbd images, when full and inc are exported,
+    # use the conf here to access them
+    nbd_image_full = nbdfull
+    nbd_image_inc = nbdinc
+    nbd_port_nbdfull = ${nbd_port_full}
+    nbd_port_nbdinc = ${nbd_port_inc}
+    nbd_export_name_nbdfull = ${nbd_export_name_full}
+    nbd_export_name_nbdinc = ${nbd_export_name_inc}
+    enable_nbd_nbdfull = yes
+    enable_nbd_nbdinc = yes
+    image_format_nbdfull = raw
+    image_format_nbdinc = raw
+
+    # conf of local backup images, copy data from
+    # nbd images into these local images by rebase
+    client_image_full = back1
+    client_image_inc = back2
+    image_size_back1 = 2G
+    image_size_back2 = 2G
+    image_format_back1 = qcow2
+    image_format_back2 = qcow2
+    image_name_back1 = back1
+    image_name_back2 = back2
+
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
+    drive_format_stg1 = vhost-user-blk-pci
+    image_vubp_props_stg1 = {"num-queues":1}
+    drive_format_back2 = ${drive_format_stg1}
+    image_vubp_props_back2 = ${image_vubp_props_stg1}
+    # Image attributes
+    qsd_image_format = {"driver":"qcow2","read-only":false,"detect-zeroes"
+    qsd_image_format += :"unmap","discard":"unmap"}
```

Comment:
add node-name here, e.g. drive_stg1

Reply:
Have modified it as your suggestion, thanks @zhencliu